website/docs/docs/user-interface/bi-directional.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Bi-Directional Contract (Preview)
+---
+
+A bi-directional contract is a test result by comparing between your provider contract (eg. OpenAPI Swagger document) and your consumer contract (pact).
+
+This section contains information of bi-directional UI on Pactflow.
+
+## Dashboard
+
+Bi-directional results are listed at dashboard. 
```

Comment:
Sounds good for me, I'll also update some other doc to align the tab name

Reply:
yeah I see in the docs that we refer to the whole page as the 'Dashboard' which is fine. It'd just when referring to things within those specific tabs it will be helpful to use the tab names